Description: Load coffeescript libraries from /usr/lib/coffeescript
 Relative path loading prevents coffee from being installed to /usr/bin
Forwarded: not-needed
Author: Geza Kovacs <gkovacs@mit.edu>
Last-Update: 2010-12-29

--- a/bin/cake
+++ b/bin/cake
@@ -2,6 +2,6 @@
 
 var path = require('path');
 var fs   = require('fs');
-var lib  = path.join(path.dirname(fs.realpathSync(__filename)), '../lib');
+var lib  = '/usr/lib/coffeescript';
 
 require(lib + '/cake').run();
--- a/bin/coffee
+++ b/bin/coffee
@@ -2,6 +2,6 @@
 
 var path = require('path');
 var fs   = require('fs');
-var lib  = path.join(path.dirname(fs.realpathSync(__filename)), '../lib');
+var lib  = '/usr/lib/coffeescript';
 
 require(lib + '/command').run();
